graphviz2drawio/models/Errors.py

```
@@ -40,3 +40,12 @@ def __init__(self, sid: str | None, gid: str | None) -> None:
         super().__init__(
             f"Missing identifiers for a geometry: sid(id): {sid}, gid(title): {gid}.",
         )
+
+
+class UnableToParseGraphError(GdValueError):
+    """Graph was unexpectedly None."""
+
+    def __init__(self) -> None:
+        super().__init__(
+            f"Graph returned by pygraphviz was unexpectedly None",
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)